### ISSUE TITLE
Compatibility with 3.7

### ISSUE THREAD

**Issue by fwojciak** [simonpercivall/astunparse#26]
Hello, it seems that astunparse is not compatible with python 3.7, because `async` is now a reserved python word.

Reproduction:
```
$ docker run --rm python:3.7 bash -c 'pip install astunparse; python -c "import astunparse"'
Collecting astunparse
  Downloading https://files.pythonhosted.org/packages/8b/ea/d38686f1718e307d83673d905dcffd142640a31a217e4e76d1de78f21b20/astunparse-1.5.0-py2.py3-none-any.whl
Requirement already satisfied: wheel<1.0,>=0.23.0 in /usr/local/lib/python3.7/site-packages (from astunparse) (0.31.1)
Collecting six<2.0,>=1.6.1 (from astunparse)
  Downloading https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Installing collected packages: six, astunparse
Successfully installed astunparse-1.5.0 six-1.11.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/astunparse/__init__.py", line 4, in <module>
    from .unparser import Unparser
  File "/usr/local/lib/python3.7/site-packages/astunparse/unparser.py", line 331
    def _generic_FunctionDef(self, t, async=False):
                                          ^
SyntaxError: invalid syntax
```

**Comment by definelicht** [comment]
This was fixed with 72bcac2fc408408c7cd07043c392fd9932714604.

**Comment by gyger** [comment]
Any change this could be released on pip to make installation easy?

**Comment by simonpercivall** [comment]
There's a number of improvements (and PRs that should be merged) that should be included in a new release. I will try to get to it, though (as anyone seeing the commit history of this project can see), I haven't had much time to do so in a while.

**Comment by CrashenX** [comment]
Workaround: `-e git+https://github.com/simonpercivall/astunparse.git#egg=astunparse # https://github.com/simonpercivall/astunparse/issues/26`

**Comment by Yoshi325** [comment]
@simonpercivall If you're having trouble finding the time to work on/release astunparse, I'm sure someone would be happy to help. Python 3.7 has been out for almost 2 months now, and I'm sure I'm not the only one looking for support for it (as evidence by the activity and +1s on this Issue).

**Comment by saily** [comment]
Could we please get a release?

**Comment by simonpercivall** [comment]
This is now (should now) be fixed in version 1.6. (Please reopen if I missed something.)
